content/en/docs/reference/glossary/object.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Object
+id: object
+date: 2020-10-12
+full_link: https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects
+short_description: >
+   A entity in the Kubernetes system, representing part of the state of your cluster.
+aka: 
+tags:
+- fundamental
+---
+An entity in the Kubernetes system. The Kubernetes API uses these entities to represent the state
+of your cluster.
+<!--more-->
+A Kubernetes object is typically a “record of intent”—once you create the object, the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} works constantly to ensure
+that the item it represents actually exists.
```

Comment:
Hi @sftim .
I agree that this term should be added.
The long definition seems to be a duplicate of the "Kubernetes API" definition.
Could you drop the last sentence and add a sentence citing some examples of objects.
The core [or fundamental] Kubernetes objects are: Pod, Deployment, Job, Volume, Secret, Service, ...
Note: The glossary categories selector lists "Core object" as: a resource type that Kubernetes supports by default.